contributing.md

```diff
@@ -0,0 +1,72 @@
+# Contributing Guide
+
+## General
+
+1. If you think something belongs in the wrong category, or think there needs to be a new category, feel free to edit things.
+1. Always add your items to the end of a list. The order is first-come-first-serve.
+1. Always add the "Add your item" to the end of each list
```

Comment:
1, 2, 3?

Reply:
MD allows (i think recommends) this. in fact, i'd use 0, 0, 0 if not for the fact that other MD parsers use the first number as the first number in the sequence. 10, 0, 0 would result in 10, 11, 12

Reply:
really? I did not know that! thanks for clearing it up! guess it's easier to add/remove items this way